test/helpers/dom/EnrollTotpManualSetupForm.js

```diff
@@ -26,7 +26,7 @@ define(['./Form'], function (Form) {
     },
 
     countryCodeSelect: function () {
-      return this.select(COUNTRY_CODE_SELECTOR);
+      return this.inputWrap(COUNTRY_CODE_SELECTOR).find('.chzn-container');
```

Comment:
Timing in our tests changed in a way that 'chosen' is rendered before we check for the existence of `countryCodeSelect`